### PR TITLE
South Lawrence scraper

### PR DESCRIPTION
**Pull request by dragthemagicpuffin**
The scraper is able to determine if there is no availability, but not if there is, since there is no working example.
I took the approach of posting the scheduling query after scraping the main site for some IDs.

### CODE REVIEW

**Comment by iann** [comment]
Hi @dragthemagicpuffin! Could you move the SouthLawrence.js file to follow the way other scrapers are structured? eg. `SouthLawrence/index.js`? Also, it looks like this PR reintroduces the `PediatricAssociatesOfGreaterSalem.js` file which is now located at `PediatricAssociatesOfGreaterSalem/index.js`.

File: data/sites.json [hunk 1]
```diff
@@ -325,5 +325,13 @@
...
             }
         ]
+    },
+    "SouthLawrenceEastElementarySchool": {
+        "name": "South Lawrence East Elementary School",
+        "website": "https://lawrencegeneralcovidvaccine.as.me/schedule.php",
+        "address": "165 Crawford Street",
+        "city": "Lawrence",
+        "state": "MA",
+        "zip": " 01843"
```
Comment (iann) [inline data/sites.json:335]: - Nitpick: leading zero on zip code. 
- Also please move this block into a `SouthLawrence/config.js`

**Review by iann — CHANGES_REQUESTED** [review]

File: data/sites.json (hunk 1 above)
Comment (dragthemagicpuffin) [reply data/sites.json:335]: - hopefully it was the leading space not 0 that was the problem :)
- moved

**Review by livgust — CHANGES_REQUESTED** [review]
Thanks @dragthemagicpuffin! this looks great and I confirmed it returns no results as expected.
One suggestion: recently we've been adding in
```
            if (!process.env.DEVELOPMENT) {
                await s3.savePageContent(`${siteName}-${loc.zip}`, page);
            }
```
or the like to help capture the page data when there _is_ availability since we don't know what that looks like exactly. Could help with being able to parse out info better when appointments start showing up.

**Review by livgust — APPROVED** [review]

**Pull request merged**